dist/Makefile.tmpl

```diff
@@ -154,7 +162,16 @@ endif
 lib/%_stubs.ml lib/%_c_stubs.c: lib_gen/%_gen.exe
 	$(DY)PATH=.:$(OPENSSL_HOME):$$$(DY)PATH $<
 
+# Compiling these files raises the pointer sign warning because the Ctypes declaration for bytes expects a char*, not a uint8_t*
+# Can be remove once ctypes is upgraded to 0.17
+lib/%_c_stubs.o: CFLAGS += -Wno-pointer-sign
```

Comment:
I already added this in the Makefiles, so you may have a conflict here when you merge, by the way

Reply:
you should remove line 80 of this makefile then